Cargo.toml

```diff
@@ -63,6 +63,8 @@ lto = true
 regex = "1"
 once_cell = "1.15"
 ignore = "0.4"
+trycmd = "0.14.4"
+snapbox = "0.4"
```

Comment:
We could have this as its own workspace member alongside `cross` and `xtask`, not needed but just a thought

Reply:
Is there any major advantage to that over faster compile times for the test suite?